lib/atat-web-api-stack.ts

```diff
@@ -85,5 +88,26 @@ export class AtatWebApiStack extends cdk.Stack {
       provisioningSfn.provisioningQueueConsumer.method,
       new apigw.LambdaIntegration(provisioningSfn.provisioningQueueConsumer.fn)
     );
+
+    const atatIdp = new idp.CognitoIdentityProvider(this, "AtatIdp", {
+      domainPrefix: `atat${environmentName.toLowerCase()}`,
+      scopes: [
+        {
+          resourceServerName: "atat",
+          scopes: [
+            { name: "read-cost", description: "Allow reading cost information" },
+            { name: "read-portfolio", description: "Allow reading information about portfolios" },
+            { name: "write-portfolio", description: "Allow creating and updating portfolios" },
+          ],
+        },
+      ],
+    });
+    const demoApp = atatIdp.addClient(
+      new idp.IdentityProviderLambdaClient(
+        "DemoClient",
+        new nodejs.NodejsFunction(this, "SampleFn", { entry: "idp/client.ts" })
```

Comment:
Is there a way to invoke this demo client without admin that i'm missing?

Reply:
There is not. And that's intentional

Reply:
Fair, I was trying to make sure it did as we expected, but believe you mentioned you got the expected log.